package.json

```diff
@@ -35,6 +35,7 @@
     "eslint": "^4.2.0",
     "mocha": "^4.0.1",
     "mockgoose": "^7.3.3",
+    "mongodb-download": "^2.2.3",
```

Comment:
can we confirm this fixes the issue on ci?

Reply:
I didn't change the travis config so it still does the preinstall for the build, this just makes life easier for local dev. but can test it out if you want?